cranelift-wasm/src/environ/dummy.rs

```diff
@@ -124,6 +125,15 @@ pub struct DummyEnvironment {
 
     /// Instructs to collect debug data during translation.
     debug_info: bool,
+
+    /// The name of the module.
+    module_name: Option<String>,
+
+    /// Function names.
+    function_names: HashMap<FuncIndex, String>,
+
+    /// Local variable names.
+    local_names: HashMap<(FuncIndex, LocalIndex), String>,
```

Comment:
added these little members to facilitate the `use_name_section` test, hopefully that isn't too much to add?